src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -56,8 +56,12 @@
   <Target Name="DiscoverEditorConfigFiles" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" BeforeTargets="CoreCompile">
     <DiscoverEditorConfigFiles InputFiles="@(Compile);@(AdditionalFile)">
       <Output TaskParameter="EditorConfigFiles" ItemName="EditorConfigFiles" />
+      <Output TaskParameter="PotentialEditorConfigFiles" ItemName="PotentialEditorConfigFiles" />
     </DiscoverEditorConfigFiles>
   </Target>
+  
+  <!-- Find and return the files paths where we looked for .editorconfig files but did not find one. -->
+  <Target Name="GetPotentialAnalyzerConfigFiles" DependsOnTargets="DiscoverEditorConfigFiles" Returns="@(PotentialEditorConfigFiles)" />
```

Comment:
Is this the name you want long-term in the project system? Is there some uniform item group we could add the EditorconfigFiles to for "existing file" tracking and the another "new file tracking" list? This would prevent the new project system from having to continually respond to changes like this.

Reply:
To my knowledge no such item groups exist, or at least we don't use them in csproj.

Responding to changes to things like this is basically the point of the project system, so I'm not too interested in trying to create a general purpose solution.